config/src/cache.rs

```diff
@@ -193,14 +193,24 @@ pub struct Cache {
 impl fmt::Display for Cache {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         for chain in &self.chains {
-            match NumberPrefix::decimal(chain.blocks.iter().map(|x| x.1).sum::<u64>() as f32) {
+            match NumberPrefix::decimal(
+                chain.block_explorer as f32 + chain.blocks.iter().map(|x| x.1).sum::<u64>() as f32,
+            ) {
                 NumberPrefix::Standalone(size) => {
                     writeln!(f, "-️ {} ({:.1} B)", chain.name, size)?;
                 }
                 NumberPrefix::Prefixed(prefix, size) => {
                     writeln!(f, "-️ {} ({:.1} {}B)", chain.name, size, prefix)?;
                 }
             }
+            match NumberPrefix::decimal(chain.block_explorer as f32) {
+                NumberPrefix::Standalone(size) => {
+                    writeln!(f, "\t-️ Block Explorer ({:.1} B)\n", size)?;
```

Comment:
Note the additional newline - it was cleaner to my eye to add an additional line to separate the etherscan data and the block data